recipe/meta.yaml

```diff
@@ -6,7 +6,7 @@ package:
 
 source:
   url: https://code.zmaw.de/attachments/download/17451/cdo-{{ version }}.tar.gz
-  sha256: cc6993e96a20d4b0829cb8f67dbc798e2ee7fa734198054e6c85bcc7d86e3f2e
+  sha256: 906e03f5518488726f8447c6ff4f7fa8
```

Comment:
sha256 is probably not that short.
Take a look at

https://github.com/conda-forge/staged-recipes#2-how-do-i-populate-the-hash-field

for some tips on how to calculate the `sha256` when that is not available upstream.